app/src/main/res/layout/story_fragment.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
```

Comment:
Nit: please remove extra space. Ditto for ``story_chapter_view.xml``.

Reply:
Done.